avro2tf/src/main/scala/com/linkedin/avro2tf/parsers/Avro2TFJobParamsParser.scala

```diff
@@ -339,6 +341,23 @@ object Avro2TFJobParamsParser {
         """Optional.
           |Whether to pass through inputs and only change outputs formats, num partitions, etc""".stripMargin
       )
+
+    opt[Seq[String]](Avro2TFJobParamNames.FEATURE_LIST_CAP)
+      .action {
+        (featureListCapStr, avro2TFParams) =>
+          val featureListCap = featureListCapStr.map { capStrEntry =>
+            val nameAndCap = capStrEntry.split(":")
```

Comment:
let's remove space first and then split to make it more robust

Reply:
I also thought about this case, a second thought is that the input string will never have space, if there is a space in user input for `featureListCapStr `,  the strings after the first space will be treated as other params, which will cause errors. So adding trim here is not going to help anything.

Reply:
no, don't do trim. Do capStrEntry.replace(" ", "").split(":"), it helps with "featurelist : 123". It's very common for people to add space in between

Reply:
my bad, what i explained above also applies to `capStrEntry.replace(" ", "").split(":")`, users can not put space in the param, the delimiter for scopt parser is space.